PacklinkPro/Helper/CsrfValidatorSkip.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Packlink\PacklinkPro\Helper;
+
+/**
+ * Class CsrfValidatorSkip
+ *
+ * @package Packlink\PacklinkPro\Helper
+ */
+class CsrfValidatorSkip
+{
+    const WHITELISTED_ACTIONS = [
+        '‌packlink_webhook_webhooks',
+        'packlink_asyncprocess_asyncprocess',
+    ];
+
+    /**
+     * Validates csrf request.
+     *
+     * @param \Magento\Framework\App\Request\CsrfValidator $subject
+     * @param \Closure $proceed
+     * @param \Magento\Framework\App\RequestInterface $request
+     * @param \Magento\Framework\App\ActionInterface $action
+     *
+     * @noinspection PhpUnusedParameterInspection*/
+    public function aroundValidate(
+        $subject,
```

Comment:
Avoid unused parameters

Reply:
It cannot be avoided when the parameter is forced by the interface. In this case, it is not clear because this class does not directly implement the interface, but Magento will call this method with the given set of parameters. Thus the unused parameter.

Reply:
ok, got it